Resources/manifest.yml

```diff
@@ -2,9 +2,4 @@
 windowIconSet: /Textures/Logo/icon
 splashLogo: /Textures/Logo/logo.png
 
-clientAssemblies:
-  - Content.Client
-  - Content.Shared
-  - Content.Shared.Database
-
```

Comment:
Is this meant to get removed, or was this just temporary to test the standalone executable? If we wanna keep the standalone exe, it'd probably be good if this just gets turned into some a whitelist that doesn't error if one of the listed assemblies is not present.

Reply:
Yeah it was intentional since we don't need the original purpose anymore. Doing the whitelist thing doesn't work for that (since it's meant to avoid people side-loading extra dlls via a content bundle).

Reply:
A better fix would be to make `Content.Replay` pass a parameter to the single player client entry point to add an extra whitelisted module that way.